### DISCUSSION TITLE
Nerfed

### DISCUSSION THREAD

**Discussion by Yoyonop** [farcasterxyz/protocol#140] [Q&A]
How to fix account warpcast nerfed? I've been waitung 4 days? 
![IMG_20240121_084031](https://github.com/farcasterxyz/protocol/assets/59782130/504c18d6-5569-49bc-9acb-eb81c3f6cd6a)

**Comment by horsefacts** [comment]
Hi there—accounts that are nerfed on Warpcast can still post to the Farcaster protocol, but do not appear on the Warpcast client. It's likely that this account violated a rule like impersonating another user or sending abusive replies and notifications.

This discussion forum is for the Farcaster protocol, so I'm going to close this conversation.
